<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatFriend</class>
 <widget class="QWidget" name="ChatFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>485</y>
     <width>700</width>
     <height>195</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>195</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>195</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_3{background:rgb(235,242,249);}</string>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#textEdit{border:none;background:rgb(235,242,249);}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{background-image:url(&quot;:/chat/emoj.png&quot;);}#label_5:hover{border:1px solid rgb(203, 203, 203);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>147</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{background-color:rgb(6,157,213);border: none;border-radius:10px;color:white;}#pushButton_2:hover{background-color:rgb(75,202,255);}</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>147</y>
      <width>95</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{background-color:rgb(6,157,213);border: none;border-radius:10px;color:white;}#pushButton:hover{background-color:rgb(75,202,255);}</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>700</width>
     <height>435</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>435</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>435</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{background:rgb(235,242,249);}</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>435</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>435</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>435</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#textBrowser{border:none;background:rgb(235,242,249);color:black;}</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{border-radius:4px;background:rgb(140,180,200);}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>5</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{font-weight:bold;font-size:20px;}</string>
    </property>
    <property name="text">
     <string>借我</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2{font-weight:bold;color:rgb(100,100,100);font-size:19px;}
#label_2:hover{background:rgb(255,255,255,50%);}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3{font-weight:bold;color:rgb(100,100,100);font-size:23px;}
#label_3:hover{background:rgb(255,255,255,50%);}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
